shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -318,6 +318,15 @@ - (BOOL)flt_hasFirstResponderInViewHierarchySubtree {
   }
 }
 
+void FlutterPlatformViewsController::PushFilterToVisitedPlatformViews(
+    std::shared_ptr<const DlImageFilter> filter) {
+  for (int64_t id : visited_platform_views_) {
+    EmbeddedViewParams params = current_composition_params_[id];
+    params.PushImageFilter(filter);
+    current_composition_params_[id] = params;
+  }
```

Comment:
We should also add a test for this part.

Reply:
I'm going to block this PR to land for now until [#34596](https://github.com/flutter/engine/pull/34596) is landed

Reply:
Offline discussion. We will land this PR first and add the final test in a follow up PR